openpype/hosts/houdini/plugins/publish/validate_primitive_hierarchy_paths.py

```diff
@@ -104,3 +113,60 @@ def get_invalid(cls, instance):
                 "(%s of %s prims)" % (path_attr, len(invalid_prims), num_prims)
             )
             return [output_node.path()]
+
+    @classmethod
+    def repair(cls, instance):
+        """Add a default path attribute Action.
+
+        it's a helper action more than a repair action,
+        which used to add a default single value.
+        """
+
+        output_node = instance.data.get("output_node")
+        rop_node = hou.node(instance.data["instance_node"])
+
+        path_attr = rop_node.parm("path_attrib").eval()
+
+        frame = instance.data.get("frameStart", 0)
+        geo = output_node.geometryAtFrame(frame)
+
+        paths = geo.findPrimAttrib(path_attr) and \
+                    geo.primStringAttribValues(path_attr)
+
+        # This check to prevent the action from running multiple times.
+        if paths:
+            return
+
+        path_node = output_node.parent().createNode("name", "AUTO_PATH")
+        path_node.parm("attribname").set(path_attr)
+        path_node.parm("name1").set('`opname("..")`/`opname("..")`Shape')
+
+        cls.log.debug(
+            "'%s' was created. It adds '%s' with a default single value"
+            % (path_node, path_attr)
+        )
+
+        path_node.setGenericFlag(hou.nodeFlag.DisplayComment,True)
+        path_node.setComment(
+            'Auto path node created automatically by "Add a default path attribute"'
+            '\nFeel free to modify or replace it.'
+        )
+
+        if output_node.type().name() in ['null', 'output']:
+            # Connect before
+            path_node.setFirstInput(output_node.input(0))
+            path_node.moveToGoodPosition()
+            output_node.setFirstInput(path_node)
+            output_node.moveToGoodPosition()
+        else:
+            # Connect after
+            path_node.setFirstInput(output_node)
+            rop_node.parm('sop_path').set(path_node.path())
+            path_node.moveToGoodPosition()
+            instance.data.update({"output_node" : path_node })
+
+            cls.log.debug(
+            "'%s' has set as the output node, "
+            "'%s' has updated"
```

Comment:
This log message doesn't seem entirely right - has set _what_ as the output node?

Reply:
maybe "'%s' has been set as the output node" ? 
I should notify artist about `sop_path` of ROP (only happens when connecting path node after)

Reply:
> SOP path on '{ropnode.path()}' updated to new output node {node.path()}

Something like that makes sense as a message?